packaging/opae/version

```diff
@@ -24,5 +24,5 @@
 # CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-declare -r PROJECT_VERSION='2.2.0'
+declare -r PROJECT_VERSION='2.3.0'
```

Comment:
Change back to 2.2.0 after this goes in..

Reply:
fixed